ccd-definition/CaseEventToFields/AcknowledgeClaim.json

```diff
@@ -22,6 +22,16 @@
     "PageDisplayOrder": 1,
     "ShowSummaryChangeOption": "N"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "ACKNOWLEDGE_CLAIM",
+    "CaseFieldID": "respondent1Details",
```

Comment:
Should it be respondent2Details?

Reply:
no, this is a CCD hack to permenantly hide a field. It can be any field